src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ CHANGELOG
  * Deprecated `Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader`, use 
    `Symfony\Component\Translation\Reader\TranslationReader` instead
  * Deprecated `translation.loader` service, use `translation.reader` instead
+ * Added `asset.request_context.base_path` and `asset.request_context.secure` parameters
```

Comment:
we could add an entry to the changelog file of the Asset component too

Reply:
and hint a bit more why they're useful :)